dataprofiler/labelers/data_processing.py

```diff
@@ -1911,25 +2003,30 @@ def priority_prediction(results, entity_priority_order):
             ]
 
     @staticmethod
-    def split_prediction(results):
+    def split_prediction(results) -> None:
         """
         Split the prediction across votes.
 
         :param results: regex from model in format: dict(pred=..., conf=...)
         :type results: dict
-        :return: aggregated predictions
+        :return: None
         """
         for i, pred in enumerate(results["pred"]):
             results["pred"][i] = pred / np.linalg.norm(
                 pred, axis=1, ord=1, keepdims=True
             )
 
-    def process(self, data, labels=None, label_mapping=None, batch_size=None):
+    def process(
+        self,
+        data: np.ndarray,
+        results: Dict,
+        label_mapping: Dict[str, int],
```

Comment:
Renamed `labels` to `results` to be consistent with other postprocessors `process` and the superclass. Also, removed default `None` from `label_mapping` because that causes an error later in the function.

Reply:
good call